### ISSUE TITLE
Show destination on hover

### ISSUE THREAD

**Issue by cabbagefork** [isaackd/AnnotationsRestored#61]
Whether it is to direct the user to a different timestamp, video, or completely unrelated site, it would be good to know beforehand. I think it would be a good QoL feature.

**Comment by isaackd** [comment]
Good idea! I agree it would be nice to know where the link is taking you.
